readme.md

```diff
@@ -0,0 +1,15 @@
+# Project Title
+
+Rogue game in Python 3 using tcod library
+
+## Description
+
+This project is about creating a rogue-like game in Python 3 using tcod library and some others. 
+
+## Features
```

Comment:
we have no features yet, I want you to leave features empty and move the following items to the todo list.